Formula/redis.rb

```diff
@@ -24,7 +24,7 @@ class Redis < Formula
   depends_on "openssl@1.1"
 
   def install
-    system "make", "install", "PREFIX=#{prefix}", "CC=#{ENV.cc}", "BUILD_TLS=yes"
+    system "make", "install", "PREFIX=#{prefix}", "CC=#{ENV.cc}", "BUILD_TLS=yes", "MALLOC=jemalloc"
```

Comment:
Does this vendor its own jemalloc? It shouldn't, and should use Homebrew jemalloc instead.

Please bump the `revision` (either add `revision 1` or increment the `revision` right after `license`) so that users who have this installed get the updated version of the formula when they do `brew upgrade`. Amend your existing commit instead of adding new ones when making changes here, please. Thanks!

Reply:
@carlocab thank you for your feedback. Hope I did amend correctly - if I made mistake please let me know and I'll close/re-create PR.

1. Added revision information
2. redis has its own/modified version of jemalloc. For example https://github.com/redis/redis/blob/unstable/deps/jemalloc/src/jemalloc.c#L3928 is improvement over original. I don't find this in upstream jemalloc code, hence I don't think using Homebrew jemalloc would render expected result.

Thanks!